moulinette/utils/filesystem.py

```diff
@@ -31,8 +31,8 @@ def read_file(file_path):
             file_content = f.read()
     except IOError as e:
         raise MoulinetteError('cannot_open_file', file=file_path, error=str(e))
-    except Exception as e:
-        raise MoulinetteError('error_reading_file', file=file_path, error=str(e))
+    except Exception:
+        raise MoulinetteError('unknown_error_reading_file', file=file_path)
```

Comment:
Eh they key did not exist ? Note that this mistake about this key is also later around line 108 ...

Ideally it would be nice to keep the `error=str(e)` in the final error message though I see that it's not used in the i18n strings so that's probably something to be careful about :/ ...

Reply:
Yep, it did not exist. Or, I only found `unknown_error_reading_file` in the locales files. I've added the `{error:s]` to the i18n string and also fixed the usages ~ line 108. That's done in the 75c8517cdef6d74e3fa3aa25ba1a4456b4cf3314 fixup commit (will rebase it all once the reviews are done).